config/Config.js

```diff
@@ -102,14 +102,14 @@ const rootDir = process.env.BEAME_DIR || path.join(home, '.beame');
 const cdrDir = process.env.BEAME_CDR_DIR || path.join(home, '.beame_cdr');
 
 /** @const {String} **/
-const scsDir = process.env.BEAME_SCS_DIR || path.join(rootDir, 'scs');
+const scsDir = process.env.BEAME_SCS_DIR || path.join(rootDir, 'ocsp-cache');
 
 /** @const {String} **/
 const remotePKsDirV1 = path.join(rootDir, 'pki');
 
 const localCertsDirV2 = path.join(rootDir, 'v2');
 
-const issuerCertsPath = path.join(rootDir, 'ocsp-cache');
+const issuerCertsPath = scsDir;
```

Comment:
Nope. Keep two directories separate. They have semantically different content.

Reply:
Imagine a code that needs to go over all items of a type. With one directory there will be ugly filtering if.

Reply:
currently the scs "store cache services" is just used for ocsp, inside that folder a file called creds.db is created. Having it called scs makes it close to impossible to understand what it does without reading the code.
The issuerCertsPath is also only used by the ocsp feature. Having it with the name ocsp-cache is also weird.

In the end, this directories will only have one file each. scs the creds.db and issuerCertsPath the beameioca1.pem.

I would say ocsp/creds.db and ocsp/issuers/beameioca1.pem  would probably move us in a good direction.
Any suggestion on a better name for them ?

Reply:
`issuer-certs-chain`